Resources/public/templates/partials/tree.hbt

```diff
@@ -1,6 +1,6 @@
 <ul class="ez-tree-level">
 {{#each children}}
-    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}">
+    <li class="ez-tree-node{{#if state.leaf}} is-tree-node-leaf{{/if}} {{#if state.open}}is-tree-node-open{{else}}is-tree-node-close{{/if}}" data-node-id="{{id}}" title="{{ data.contentInfo.name }}">
```

Comment:
kind of the same as https://github.com/ezsystems/PlatformUIBundle/pull/653/files#r72211802, but there's already a `data-node-id` attribute which contains the Location id, I think, you should use it instead of search by Content name.